Nodejs/Setup/swix/Microsoft.VisualStudio.NodejsTools.Targets.swixproj

```diff
@@ -31,7 +31,9 @@
   </Target>
 
   <PropertyGroup> 
-    <PackagePreprocessorDefinitions>$(PackagePreprocessorDefinitions);BuildOutputRoot=$(BuildOutputRoot)</PackagePreprocessorDefinitions>
+    <WebRoleDllPath Condition="'$(SignedBinariesPath)' != ''">$(SignedBinariesPath)</WebRoleDllPath>
+    <WebRoleDllPath Condition="'$(SignedBinariesPath)' == ''">$(BuildOutputRoot)\Binaries\Nodejs\Debugger\RemoteDebug</WebRoleDllPath>
```

Comment:
nit: including an extra backlash in folder definitions is helpful so that it's easier to append paths